Classes/Domain/Model/Index.php

```diff
@@ -191,9 +191,9 @@ public function request(string $method, string $path = null, array $arguments =
      */
     public function create(): void
     {
-        $indexConfiguration = $this->getConfiguration();
-        $indexCreateObject = array_filter($indexConfiguration, static fn($key) => in_array($key, self::$allowedIndexCreateKeys, true), ARRAY_FILTER_USE_KEY);
-        $this->request('PUT', null, [], json_encode($indexCreateObject));
+        $indexConfiguration = $this->getConfiguration() ?? [];
+        $indexCreateObject = is_array($indexConfiguration) ? array_filter($indexConfiguration, static fn($key) => in_array($key, self::$allowedIndexCreateKeys, true), ARRAY_FILTER_USE_KEY) : [];
```

Comment:
Isn't that `is_array()` check useless now, as in the line before it's made into an empty array if `null`?

Reply:
It's, fixing